sdks/java/core/src/main/java/org/apache/beam/sdk/testing/BigqueryMatcher.java

```diff
@@ -121,8 +121,8 @@ protected boolean matchesSafely(PipelineResult pipelineResult) {
       throw new RuntimeException("Failed to fetch BigQuery data.", e);
```

Comment:
I am not familiar with this framework.

However, looks to me describeMismatchSafely depends on matchesSafely to set response.
And, if RuntimeException is thrown in here, would that breaks any contracts?